samples/ArtistSite/ArtistSite.App/Pages/ArtistDetail.razor.cs

```diff
@@ -10,7 +10,7 @@ public string ProfileImageUrl
         {
             get
             {
-                return $"Images/Profile/{Artist.ArtistId}.jpg";
+                return $"artist-profile/{Artist.ArtistId}.png";
```

Comment:
Same comment about not assuming/hard-coding a file extension here...probably should be data-driven and not rely on images matching a primary-key value followed by .png (too rigid).